modules/components/tag-input.ts

```diff
@@ -802,7 +802,6 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
     private setUpOnBlurSubscriber(): void {
         this.inputForm
             .onBlur
-            .filter(() => this.dropdown && this.dropdown.isVisible)
```

Comment:
Ok, I see what it's for now. I added the filter and a test around addOnBlur and dropdown.isVisible.